gpytorch/means/constant_mean_grad.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+
+import torch
+from .mean import Mean
+
+
+class ConstantMeanGrad(Mean):
```

Comment:
Is there a way that we could make this more general - like how `MultitaskMean` wraps arbitrary means?

Reply:
I'm not sure if this is possible. The mean vector evaluated at one point is a vector of length d+1 with the original scalar mean function and the corresponding d partial derivatives along x_1, ..., x_d. I think this class has to be implemented for each mean, just like ConstantMean etc.

Reply:
Okay that should be fine then.